app/server-control/directives/kvm-console.js

```diff
@@ -30,11 +30,18 @@ window.angular && (function(angular) {
                 return false;
               };
 
+              var KVMconn = 0;
```

Comment:
I think it would be more normal to do a boolean here. 
var KVMconn = false;

KVMconn = true;

and then 

if (KVMconn === true)

or 

if (KVMconn)

Read: https://stackoverflow.com/questions/15393935/boolean-in-an-if-statement/15394481

Reply:
I will revise it, thank you for sharing information and suggestions.